test/src/test/crypto.h

```diff
@@ -16,7 +16,7 @@ static bool validate_aesgcm()
 	char *decrypted = calloc(1, strlen(plain) + 1);
 	char *tag1 = calloc(1, 16);
 
-	MTY_AESGCM *aes1 = MTY_AESGCMCreate(password1);
+	MTY_AESGCM *aes1 = MTY_AESGCMCreate(password1, false);
```

Comment:
Why is this called with false, shouldn't this be called with 16? Same with the other ones.